### ISSUE TITLE
Accessibility status user list column should be sortable

### ISSUE THREAD

**Issue by admturner** [washingtonstateuniversity/WSUWP-Plugin-A11y-Status#28]
It would be helpful to be able to sort the user list table by accessibility status to easily see all users that are not certified.
